js/locale/locale.js

```diff
@@ -72,6 +72,7 @@ export class LocalePlugin extends RipeCommonsPlugin {
                 this.setLocaleValue(`${prefix}${key}`, bundle[key], locale);
             }
         }
+        this.owner.trigger("localeMap_updated");

```

Comment:
rename the event to `locale_map_changed` to be consistent with the `locale_changed` one